mage/query-modules/python/elasticsearch-synchronization.md

```diff
@@ -0,0 +1,315 @@
+---
+id: elasticsearch-synchronization
+title: elasticsearch_synchronization
+sidebar_label: elasticsearch_synchronization
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import RunOnSubgraph from '../../templates/_run_on_subgraph.mdx';
+
+export const Highlight = ({children, color}) => (
+<span
+style={{
+      backgroundColor: color,
+      borderRadius: '2px',
+      color: '#fff',
+      padding: '0.2rem',
+    }}>
+{children}
+</span>
+);
+
+<!--[![docs-source](https://img.shields.io/badge/source-elasticsearch_synchronization-FB6E00?logo=github&style=for-the-badge)]("") -->
+
+## Abstract
+**Elasticsearch** is a **text-processing platform** that can be used to enhance the capabilities of a graph database like Memgraph. It offers many fine-grained features useful when working on a text that is impossible to develop in databases. Data residing in Elasticsearch and Memgraph should be **synchronized** because otherwise, the whole system could be in an inconsistent state. Such a feature can be added inside Memgraph by using triggers: every time a new entity is added (node or edge) it gets indexed to the Elasticsearch index.
+
+
+### Blog Posts
+
+The following blog posts explain how Memgraph integrated Elasticsearch into its technology stack:
+<!--- [Elasticsearch](TODO add URL when blog post will be published)-->
```

Comment:
Add this as a label, look at this page for an example (the purple label about blog posts -> https://memgraph.com/docs/memgraph/reference-guide/replication)

Reply:
Will do it once the blog post is published